func main() {
var x;
var y;
x = foo(3); 
y = x + 10;
print(y);
print(x); /* foo(3) is NOT re-evaluated */
}


/*
*OUT*
ErrorType.NAME_ERROR
*OUT*
*/